python/paddle/fluid/framework.py

```diff
@@ -1828,6 +1828,9 @@ def __init__(self, real, imag):
             self._dtype = "complex128"
         self._shape = self.real.shape
 
+    def __getitem__(self, idx):
```

Comment:
slice contains multiple case, here we can refer to the implementation of Variable

https://github.com/PaddlePaddle/Paddle/blob/68449d19a5ad0fc31220087f15ef438f213d58ed/python/paddle/fluid/framework.py#L1760

Reply:
after discussion， can reuse the slice implement of tensor